components/Header.js

```diff
@@ -1,32 +1,29 @@
-import React, { useEffect, useRef, useState } from "react";
+"use client";
+import React, { useEffect, useState, useRef } from "react";
 import Image from "next/image";
-import styles from "../styles/LoginDropdown.module.css";
-import router from "next/router";
-import { useAuth } from "../context/AuthContext";
-import DropdownCard from "../components/LoginDropdown";
+import Link from "next/link";
+import { RxHamburgerMenu } from "react-icons/rx";
+import DropdownCard from "./LoginDropdown";
+import { AiOutlineSearch } from "react-icons/ai";
+import { SunIcon } from "@heroicons/react/24/solid";
+import logo from "../public/logo.png";
 
-function Header({ navbarBackground }) {
-  // localStorage.setItem("redirectUrl", window.location.href);
-  // const [isLoggedIn, setLoggedIn] = useState(false);
+function Header({ isUserLoggedIn }) {
```

Comment:
we are using `Authcontext` for managing  auth state So i think we do not need to pass here!

Reply:
done